frontend/src/modules/dealerships/DealershipLocations.vue

```diff
@@ -0,0 +1,345 @@
+<template>
+  <div>
+    <CCard class="mt-2">
+      <CCardBody>
+        <h3 class="mb-3">Dealership Zones</h3>
+        <CRow>
+          <CCol v-if="addingNewZone">
+            <CForm @submit.prevent="submitNewZone" v-if="addingNewZone">
+              <CInput
+                label="Name"
+                placeholder="Unique name for the zone"
+                v-model="newZoneName"
+                required
+              />
+              <CTextarea
+                label="Description (optional)"
+                placeholder="Enter the zone's description"
+                rows="5"
+                v-model="newZoneDescription"
+              />
+              <CButton
+                color="secondary"
+                @click="
+                  () => {
+                    path = [];
+                    addingNewZone = false;
+                  }
+                "
+                >Cancel</CButton
+              >
+              <CButton color="primary" type="submit">Add Zone</CButton>
+              <CButton color="danger" @click="undoLastPoint">Undo</CButton>
+            </CForm>
+          </CCol>
+          <CCol>
+            <GmapMap
+              :center="center"
+              :zoom="17"
+              ref="mapRef"
+              map-type-id="satellite"
+              style="height: 500px"
+              :options="{
+                streetViewControl: false,
+                fullscreenControl: false,
+                rotateControl: false,
+              }"
+              @click="mapClick"
+            >
+              <!-- <gmap-polyline
+                v-if="newPath.length > 1 && newPath.length < 4"
+                ref="polylineRef"
+                :path="newPath"
+                :editable="false"
+                :visible="true"
+                :options="{ fillColor: 'red', fillOpacity: 0.5 }"
+              /> -->
+              <gmap-polygon
+                v-if="newPath.length > 0"
+                ref="polygonRef"
+                :path="newPath"
+                :editable="false"
+                :visible="true"
+                :options="{ fillColor: 'red', fillOpacity: 0.5 }"
+                @click="mapClick"
+              />
+              <gmap-polygon
+                v-for="zone in zones"
+                :key="zone._id"
+                :path="zone.path"
+                :editable="false"
+                :visible="true"
+                :draggable="true"
+                :options="{
+                  fillColor: zone.fillColor,
+                  fillOpacity: 0.5,
+                  strokeWeight: 0,
+                  strokeColor: 'blue',
```

Comment:
These seem redundant options and can be removed

Reply:
Addressed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/87/commits/b3016ee9df8271d80dc6fc49195c95de810e1398)